src/_blog/libp2p-IPFSþing-2023-recap.md

```diff
@@ -0,0 +1,137 @@
+---
+tags:
+- libp2p
+title: libp2p at IPFSþing 2023 Recap
+description: Recap of libp2p presentations and workshops at IPFSþing 2023
+date: 2023-05-05
+permalink: "/2023-05-05-libp2p-IPFS%C3%BEing-recap/"
+translationKey: ''
+header_image: /ipfs-thing-2023-logo.svg
```

Comment:
For some reason the header image is cutoff in the preview here https://bafybeido4ueaawkfbfw5h5dmd3327kumgjdupl5qmd3jc7nrybdnpem6wu.on.fleek.co/

looks fine in the actual article though

Reply:
It's because the image is an .svg. I'll render it to a .png and update it.

Reply:
fixed in the new version I'm about to push